neo/Network/P2P/TaskSession.cs

```diff
@@ -12,7 +12,7 @@ internal class TaskSession
         public readonly IActorRef RemoteNode;
         public readonly VersionPayload Version;
         public readonly Dictionary<UInt256, DateTime> Tasks = new Dictionary<UInt256, DateTime>();
-        public readonly HashSet<UInt256> AvailableTasks = new HashSet<UInt256>();
+        public readonly HashSet<UInt256> InvBlockPendingTasks = new HashSet<UInt256>();
```

Comment:
Could you add comments to this vars also @vncoelho ? (summary)

Reply:
Yes, makes sense.